core-war/src/main/java/org/silverpeas/web/pdc/control/PdcSearchSessionController.java

```diff
@@ -573,18 +607,21 @@ private void processFacetsFormField(Map<String, Facet> fieldFacetsMap, GlobalSil
     return fieldFacets;
   }
 
-  private String getFieldLabel(String formName, String fieldName)
+  private FieldTemplate getFieldTemplate(String formName, String fieldName)
       throws PublicationTemplateException, FormException {
     PublicationTemplate form =
         PublicationTemplateManager.getInstance().loadPublicationTemplate(formName);
-    return form.getRecordTemplate().getFieldTemplate(fieldName).getLabel(getLanguage());
+    return form.getRecordTemplate().getFieldTemplate(fieldName);
+  }
+
+  private String getFieldLabel(String formName, String fieldName)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused private "getFieldLabel" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AUnusedPrivateMethod)